soh/soh/Enhancements/randomizer/randomizer_check_objects.h

```diff
@@ -24,6 +24,7 @@ typedef struct {
     bool visibleInImgui;
     std::string rcShortName;
     std::string rcSpoilerName;
+    bool vanillaCheck;
```

Comment:
I wonder if there's a better name for this.

Reply:
I'm open to suggestions. The name merely came from the idea of a vanilla (playthrough) check. If we want more verbosity in the naming, then we could just do something like `vanillaRunCheck`?

Reply:
Is the intention for this mainly to be used for tracking progress of a hundo run?

Reply:
Yes, the checks that are `true` for that variable are items that would be expected in a 100% vanilla run, excepting the masks since we don't have those as checks in rando.

Reply:
How about `vanilla100Check` or `vanillaHundoCheck`

Reply:
`vanillaHundoCheck` it is.